data/json/items/ammo/sling-ready_grenade.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "type": "AMMO",
+    "id": "sling-ready_molotov",
+	"name": { "str": "sling-ready Molotov cocktail" },
+    "description": "A bottle of flammable liquid with a long fuse inserted ready to be used in a sling.  Use this item to light the fuse.  You will, of course, need a lighter or matches in your inventory to do this.  After lighting it, throw it to cause fires.",
+    "weight": "742 g",
+    "volume": "750 ml",
+    "price": 500,
+    "price_postapoc": 500,
+    "to_hit": 1,
+    "bashing": 5,
+    "material": [ "glass", "cotton" ],
+    "symbol": "*",
+    "color": "light_gray",
+    "initial_charges": 1,
+    "max_charges": 1,
+    "use_action": {
+      "need_wielding": true,
+      "target": "molotov_lit",
+      "msg": "You light the Molotov cocktail!",
+      "target_charges": 1,
+      "active": true,
+      "moves": 75,
+      "need_fire": 1,
+      "menu_text": "Light rag",
+      "type": "transform"
+    },
+    "flags": [ "NPC_ACTIVATE", "NO_REPAIR" ]
```

Comment:
```suggestion
    "flags": [ "NPC_ACTIVATE", "NO_REPAIR" ],
```

Reply:
Thank you, have corrected in the latest commit.